app/htmls/SceneBrowser.html

```diff
@@ -60,12 +60,13 @@
     }
 
     div.stream_text {
-      position: absolute;
-      left: 5px;
+      position: relative;
+      /* left: 5px;
       bottom: 5px;
-      right: 20%;
-      background-color: rgba(0, 0, 0, 0.5);
+      right: 20%; */
```

Comment:
Remove code that have been commented out, no need to keep it in the file since we have the old code in Git if needed.